### ISSUE TITLE
Please Update readme.md

### ISSUE THREAD

**Issue by Mannshoch** [microg/UnifiedNlp#207]
**Describe the bug**
The current readme contain a link to releases with last Version is from 2017
Also the current readme Miss information about Android newe than 7.

**To Reproduce**
1. Go to 'https://github.com/microg/UnifiedNlp
2. Read the readme

**Expected behavior**
 - Some information about Android New er than 7
  -  Is there may a Magisk module for that?
 - Link to newer release

**System**
Android Version: 10
Custom ROM: LineageOS 17.1

**Additional context**
After installing unified nlp from fdroid itry to find a way to register it in system and ended here.

**Comment by nobodywasishere** [comment]
https://blog.eowyn.net/unifiednlp/

**Comment by Mannshoch** [comment]
For the record, @nobodywasishere your solution worked. 

At moment, I'm wondering why there are so many old Issues without progress or get closed becasue not anymore relevant and why the readme.md is still on that old state.

**Comment by DJCrashdummy** [comment]
@nobodywasishere thank you for the clear guide, but may i suggest to add a hint, that Magisk **must not** be installed to get UnifiedNlp working.
it took me hours to figure it out and sadly on LineageOS 17.1 it seems to be the case, that it also stops working if you install Magisk afterwards.

does anybody have an idea how to get UnifiedNlp working with a rooted LineageOS?

**Comment by IzzySoft** [comment]
By flashing microG? It's part of the core. Works for me, at least.

**Comment by DJCrashdummy** [comment]
@IzzySoft yes i know his would be kind of a workaround... but i prefer a plain (rooted) LineageOS without any of the Google or microG shenanigans which i have to keep an eye on, that everything i don't like is deactivated.

...not to mention, that my device gets at *LineageOS for microG* an update once in ~4 months and the plain LineageOS once every ~2 weeks. :unamused:

**Comment by IzzySoft** [comment]
OK, the update frequency is an argument. As for the shenanigans: I simply leave them disabled, which is the default anyway. But as you know of this, I'll shut up – mentioned it just in case :wink:

**Comment by DJCrashdummy** [comment]
> As for the shenanigans: I simply leave them disabled, which is the default anyway.

@IzzySoft i hate it that i have to disagree with you, but at the laste devices i flashed with "LineageOS for microG", the "Google device registration" at the "microG Settings" was on by default.
i have to admit, that i have hardly an idea what it does... but i know that i don't need and don't want it!

**Comment by IzzySoft** [comment]
@DJCrashdummy to take away your hatred: there was a bug with that. Even after an update, on some devices it was turned on again. But that bug was fixed months ago, and I never saw it again after that. See: https://github.com/microg/GmsCore/issues/1554
